include/plugin_interface/SC_Graph.h

```diff
@@ -35,6 +35,8 @@ struct Graph
 	uint32 mNumControls;
 	float *mControls;
 	float **mMapControls;
+	int32 *mAudioBusOffsets;
+	uint32 mAudioBusOffset;
```

Comment:
this appears to have been accidentally reintroduced - could you please remove it?